evennia/contrib/worldloader/example_tutorial_world/worlddata/models.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+
+
+#------------------------------------------------------------
+#
+# store all rooms
+#
+#------------------------------------------------------------
+class world_rooms(models.Model):
+    "Store all unique rooms."
+
+    key = models.CharField(max_length=255, primary_key=True)
+    name = models.CharField(max_length=255)
+    alias = models.CharField(max_length=255, blank=True)
+    typeclass = models.CharField(max_length=255)
+    desc = models.TextField(blank=True)
+    location = models.CharField(max_length=255, blank=True)
+    home = models.CharField(max_length=255, blank=True)
+    lock = models.CharField(max_length=255, blank=True)
+    attributes = models.TextField(blank=True)
+    tutorial_info = models.TextField(blank=True)
```

Comment:
This database field seems very tutorial-world specific to belong in a full app. Or is `tutorial_info` intended as a general feature for learning a game?